### PR TITLE
feat(docs): New repair and statistics pages

### PR DESCRIPTION
**Pull request by aviv926**
This PR adds information to the documentation on the repair page and the statistics page. In addition, I added some tips that may be useful.

### CODE REVIEW

**Comment by aviv926** [comment]
It failed due to a future change to the file
`remove-offline-files.md` which will become` remove-offline-paths.md`
To specify the purpose of the page.

File: docs/docs/administration/Repair-page.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Natural State
+
+In this situation, everything is in its place and there is no problem that the system administrator should be aware of.
+
+<img src={require('./img/Repair-page.png').default} title="server statistic" />
+
+## Any Other Situation
+
+:::note ram usge
```
Comment (danieldietzler) [inline docs/docs/administration/Repair-page.md:13]:
```suggestion
:::note RAM Usage
```

File: docs/docs/administration/Repair-page.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Several users report a situation where the page fails to load, in order to solve this problem you should try to allocate more RAM for Immich, if the problem continues, you should stop using the reverse proxy while loading the page.
```
Comment (danieldietzler) [inline docs/docs/administration/Repair-page.md:14]:
```suggestion
Several users report a situation where the page fails to load. In order to solve this problem you should try to allocate more RAM to Immich, if the problem continues, you should stop using the reverse proxy while loading the page.
```

File: docs/docs/administration/Repair-page.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+:::
+
+Any other state describes the following 3 states:
```
Comment (danieldietzler) [inline docs/docs/administration/Repair-page.md:17]: This sentence doesn't make sense to me.

File: docs/docs/administration/Repair-page.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+- MATCHES - These files are matched by their checksums.
+
+- OFFLINE PATHS - These files are the results of manual deletion of the default upload library.
```
Comment (danieldietzler) [inline docs/docs/administration/Repair-page.md:21]:
```suggestion
- OFFLINE PATHS - These files are the result of manually deleting files in the upload library or a failed file move in the past (losing track of a file).
```

File: docs/docs/administration/Repair-page.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+:::tip
+To get rid of Offline paths you can follow this [guide](/docs/guides/remove-offline-paths.md)
+:::
+
+- UNTRACKS FILES - These files are not tracked by the application. They can be the results of failed moves, interrupted uploads, or left behind due to a bug.
```
Comment (danieldietzler) [inline docs/docs/administration/Repair-page.md:27]: That those are still called "untracks" is a really bad typo lol. Let me go fix that real quick

Comment (danieldietzler) [reply docs/docs/administration/Repair-page.md:27]:
```suggestion
- UNTRACKS FILES - These files are not tracked by the application. They can be the result of failed moves, interrupted uploads, or left behind due to a bug.
```

Comment (danieldietzler) [reply docs/docs/administration/Repair-page.md:27]: #8051

**Comment by aviv926** [comment]
Thanks for the feedback, corrected accordingly.

**Review by alextran1502 — APPROVED** [review]

File: docs/docs/administration/Repair-page.md (hunk 3 above)
Comment (danieldietzler) [reply docs/docs/administration/Repair-page.md:17]: I still don't understand what you're trying to say with this @aviv926

Comment (danieldietzler) [reply docs/docs/administration/Repair-page.md]: Ah thanks!

**Review by danieldietzler — APPROVED** [review]
Thank you! And sorry for not understanding that one line lol

**Pull request merged**